src/imports.jl

```diff
@@ -82,6 +82,7 @@ function add_to_imported_modules(scope::Scope, name::Symbol, val)
     end
 end
 no_modules_above(s::Scope) = !CSTParser.defines_module(s.expr) || s.parent === nothing || no_modules_above(s.parent)
+function get_named_toplevel_module(s, name) end
```

Comment:
```suggestion
function get_named_toplevel_module(s, name) 
    return nothing
end
```
Nitpick, but I'd prefer being explicit about this.

Reply:
Yep thats fine